qa/standalone/ceph-helpers.sh

```diff
@@ -160,36 +160,11 @@ function teardown() {
         && [ $(stat -f -c '%T' .) == "btrfs" ]; then
         __teardown_btrfs $dir
     fi
-    local cores="no"
-    local pattern="$(sysctl -n $KERNCORE)"
-    # See if we have apport core handling
-    if [ "${pattern:0:1}" = "|" ]; then
-      # TODO: Where can we get the dumps?
-      # Not sure where the dumps really are so this will look in the CWD
-      pattern=""
-    fi
-    # Local we start with core and teuthology ends with core
-    if ls $(dirname $pattern) | grep -q '^core\|core$' ; then
-        cores="yes"
-        if [ -n "$LOCALRUN" ]; then
-	    mkdir /tmp/cores.$$ 2> /dev/null || true
-	    for i in $(ls $(dirname $(sysctl -n $KERNCORE)) | grep '^core\|core$'); do
-		mv $i /tmp/cores.$$
-	    done
-        fi
-    fi
-    if [ "$cores" = "yes" -o "$dumplogs" = "1" ]; then
+    if [ "$dumplogs" = "1" ]; then
```

Comment:
the `EXPECT_DEATH` tests should not create a core file. the last of them who were dumping cores were fixed by 6beaf0b5d4d6e6fe54e8076019ac2303e6f1ad05.

Reply:
@liewegas @tchaikov 
I have not find the time to actually make the `EXPECT_DEATH tests` not dump cores.
So  although I agree with you, factual this is not the case on FreeBSD.
Perhaps excluding `*unittest*` cores while grepping?

Reply:
@liewegas @wjwithagen since #17447 has been merged. can we drop this commit?

Reply:
@tchaikov 
I think you also have to ask @dzafman, since he is the original submitter of this part.
The problem with this fix I had is amended by #17447, because there are no more cores.
And I'm very much in favor of making  dumplogs optional, since it really bloats output, also in cases where there are other means of capturing the logs..